afs/afs-mapdb-storage/src/main/java/com/powsybl/afs/mapdb/storage/MapDbAppStorage.java

```diff
@@ -188,18 +192,32 @@ protected MapDbAppStorage(String fileSystemName, Supplier<DB> db) {
         return map;
     }
 
-    private static <K, V> Map<K, List<V>> removeFromList(Map<K, List<V>> map, K key, V value) {
+    private static <K, V> boolean removeFromList(Map<K, List<V>> map, K key, V value) {
```

Comment:
Is there a good reason to remove the item from a copy of the list (or the set) and to put this copy in the map instead of removing the value directly from the original list (or set)?